src/main/java/uk/gov/hmcts/reform/userprofileapi/oidc/JwtGrantedAuthoritiesConverter.java

```diff
@@ -52,6 +52,9 @@ public Collection<GrantedAuthority> convert(Jwt jwt) {
         return authorities;
     }
 
+    @SuppressWarnings("java:S6204")
+    // Causes breaking issue with type of list being returned
+    // "Required type: List <GrantedAuthority>, Provided: List <SimpleGrantedAuthority>"
```

Comment:
are these comment required?

Reply:
@ishratt ideally if we are adding a Suppression tag we should add information as to why

Reply:
that makes sense, thank you